pkg/common/utils/resources.go

```diff
@@ -181,24 +181,31 @@ func (r *Resources) getNamespaceQuotas(ctx context.Context, c client.Client, ins
 
 // checkAgainstResources compares this resources against the given resources.
 // Any non-populated fields are checked against their defaults.
-func (r *Resources) checkAgainstResources(quotaResources Resources) field.ErrorList {
+func (r *Resources) checkAgainstResources(quotaResources Resources, gk schema.GroupKind, instances int64) field.ErrorList {
 	foundErrs := field.ErrorList{}
 
 	errCPURequests := field.Forbidden(r.CPURequestsPath, "Max allowed CPU requests: "+DefaultCPURequests.String()+". Configured requests: "+r.CPURequests.String()+". "+contactSupportMessage)
+
+	rDef := GetDefaultResources()
+
+	if gk.Kind == "VSHNPostgreSQL" {
```

Comment:
I'm still a bit triggered by this, but I think for now it's okay.

We can have another looks at this if/when we have more services with sidecars. (probably when we add the exporter to redis)

Reply:
not perfect, but easy to append, I'd just leave it as it is, if works correctly